earth_enterprise/src/fusion/portableglobe/cutter/cgi-bin/geecheck_tests/user_tests/disk_space_test.py

```diff
@@ -73,7 +73,7 @@ class TestDiskSpace(unittest.TestCase):
     @unittest.skipUnless(common.IsFusionInstalled(), 'Fusion is not installed')
     def testAdequateDiskSpace(self):
       """Check that the remaining disk space is at least 20%."""
-      self.assertLessEqual(20, getDiskInfo())
+      self.assertLessEqual(20, getDiskFreeSpace())
```

Comment:
is `assertLessEqual` doing what the comment saying ? or it needs to be `assertGreatEqual` if such function exist ?

Reply:
This checks that 20 <= getDiskFreeSpace and fails otherwise, so the free space must be greater than or equal to 20% in order for the test to pass

Reply:
ah ok. just the order was not clear to me.